### PR TITLE
Varvara/feature/navigation

### PR DESCRIPTION
**Pull request by varvarashcherbyna7**

### CODE REVIEW

File: App.tsx [hunk 1]
```diff
@@ -1,46 +1,54 @@
...
-    backgroundColor: isDarkMode ? Colors.darker : Colors.lighter,
-  };
+function App(): JSX.Element {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [isSignedIn, setIsSignedIn] = useState<boolean>(false);
 
+  // if (isLoading) {
+  //   // We haven't finished checking for the token yet
+  //   return <SplashScreen />;
+  // }
```
Comment (StoliarskyiKostiantyn) [inline App.tsx:27]: needed?

File: src/components/WrapperRegistrationLogin.tsx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+  },
+  footerText: {
+    ...palette.text,
+    fontWeight: '600',
+    fontSize: 14,
+  },
+  footerHelperText: {
+    color: colors.helperText,
+  },
+});
```
Comment (StoliarskyiKostiantyn) [inline src/components/WrapperRegistrationLogin.tsx:99]: You going to keep styles in component files? or maybe better move them to separate file?

File: src/components/common/btn/Btn.tsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import {Text, TouchableOpacity} from 'react-native';
+
+interface IBtn {
+  stylesBtn: any;
+  textBtn: any;
```
Comment (StoliarskyiKostiantyn) [inline src/components/common/btn/Btn.tsx:6]: is there better way to do this?

File: src/screens/LogIn.tsx [hunk 4]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import strings from '../intl18/strings';
+// import palette from '../assets/palette';
```
Comment (StoliarskyiKostiantyn) [inline src/screens/LogIn.tsx:3]: needed?

File: src/screens/LogIn.tsx [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+      footerText2={strings.login.register}
+      handlePressFooterText2={() => Alert.alert(strings.login.donotAccount)}>
+      <Text>Log In</Text>
+    </WrapperRegistrationLogin>
+  );
+};
+
+export default LogIn;
+
+// const styles = StyleSheet.create({});
```
Comment (StoliarskyiKostiantyn) [inline src/screens/LogIn.tsx:28]: same

File: src/screens/Registration.tsx [hunk 6]
```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import strings from '../intl18/strings';
+// import palette from '../assets/palette';
```
Comment (StoliarskyiKostiantyn) [inline src/screens/Registration.tsx:3]: needed?

File: src/screens/Registration.tsx [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+        Alert.alert(strings.registration.alreadyAccount)
+      }>
+      <Text>Registration</Text>
+    </WrapperRegistrationLogin>
+  );
+};
+
+export default Registration;
+
+// const styles = StyleSheet.create({});
```
Comment (StoliarskyiKostiantyn) [inline src/screens/Registration.tsx:30]: same

**Review by StoliarskyiKostiantyn — APPROVED** [review]

File: src/screens/WelcomeTenKabel.tsx [hunk 8]
```diff
@@ -0,0 +1,113 @@
+import React, {useState} from 'react';
+import {StyleSheet, Text, View} from 'react-native';
+import WrapperOnboarding from '../components/WrapperOnboarding';
+import strings from '../intl18/strings';
+import CardNameProfession from '../components/CardNameProfession';
+import images from '../assets/images';
+
+const testDataProfession = [
```
Comment (Taraslysak) [inline src/screens/WelcomeTenKabel.tsx:8]: What id type of `testDataProfession` ?

**Review by Taraslysak — CHANGES_REQUESTED** [review]

File: src/assets/images.ts [hunk 9]
```diff
@@ -4,6 +4,13 @@ const images = {
   login: require('./images/login.png'),
   googleLogo: require('./images/google-logo.png'),
   confirmPhone: require('./images/confirm_phone.png'),
+  building: require('./images/profession/building.png'),
+  // profession
+  electricity: require('./images/profession/electricity.png'),
+  multiple: require('./images/profession/multiple.png'),
+  name3: require('./images/profession/name3.png'),
```
[the remark this answers is not in the document]
Comment (varvarashcherbyna7) [reply src/assets/images.ts:11]: .) I will change this, thanks

File: src/screens/WelcomeTenKabel.tsx (hunk 8 above)
Comment (varvarashcherbyna7) [reply src/screens/WelcomeTenKabel.tsx:8]: Thanks for the comments, it is very important for me!) I will fix everything according to the recommendations

File: src/screens/Registration.tsx (hunk 7 above)
Comment (varvarashcherbyna7) [reply src/screens/Registration.tsx]: Thanks for the comments) I will fix everything according to the recommendations

**Pull request merged**